uint8[] rgb_values
---
bool success
